installer/templates/phx_ecto/repo.ex

```diff
@@ -6,6 +6,10 @@ defmodule <%= app_module %>.Repo do
   DATABASE_URL environment variable.
   """
   def init(_, opts) do
-    {:ok, Keyword.put(opts, :url, System.get_env("DATABASE_URL"))}
+	  if url = System.get_env("DATABASE_URL") do
+	    {:ok, Keyword.put(opts, :url, url)}
+	  else
+	    {:ok, opts}
+	  end
```

Comment:
LGTM, I also had to modify it in my project ...maybe you could just fix the indentation to spaces?

Reply:
I've reviewed the code formatting, should be good now